index.js

```diff
@@ -374,6 +480,36 @@ function toggleSidebar() {
 }
 
 /**
+* Shows the user data sidebar which provides the user with a button to
+* open the newtab page, reset the auto-import, and view a preview of the data that was imported
+*/
+function showUserDataSidebar() {
+    store('lastSidebarLaunchTime', Date.now());
+
+    content = sidebar.Sidebar({
+        id: 'allboard-importdata',
+        title: 'We\'ve got your back.',
+        url: './tmpl/show_data.html',
+        onAttach: function(worker) {
+            getHistory(worker);
+            getBookmarks(worker);
+        },
+        onDetach: function() {
+            content.dispose();
+            // starts the timer that will call showBadge and queue up the next
+            // sidebar to be shown. Only start the timer if there is not one
+            // already scheduled.
```

Comment:
update comment to reflect this use case or, you can leave it off here if you want.